crates/biome_css_analyze/tests/specs/nursery/noDuplicateProperties/invalid.css.snap

```diff
@@ -0,0 +1,318 @@
+---
+source: crates/biome_css_analyze/tests/spec_tests.rs
+expression: invalid.css
+---
+# Input
+```css
+a {
+    color: pink;
+    color: orange;
+}
+
+a {
+    color: pink;
+    color: pink;
+    color: pink;
+}
+
+a {
+    color: pink;
+    color: pink;
+    color: orange;
+}
+
+a {
+    color: pink;
+    background: orange;
+    color: orange;
+}
+
+a {
+    color: pink;
+    background: orange;
+    background: pink;
+}
+
+a { color: pink; { &:hover { color: orange; color: black; } } }
+
+a { color: pink; @media { color: orange; color: black; } }
+
+@media { color: orange; .foo { color: black; color: white; } }
+
+a { color: pink; @media { color: orange; &::before { color: black; color: white; } } }
+
+a { color: pink; @media { color: orange; .foo { color: black; color: white; } } }
+
+a { -webkit-border-radius: 12px; -webkit-border-radius: 10px; }
+
+a { color: red !important; color: blue; }
+
+a { color: red !important; color: blue !important; }
+
+```
+
+# Diagnostics
+```
+invalid.css:2:17 lint/nursery/noDuplicateProperties ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
+
+  ! Duplicate properties are not allowed.
+  
+    1 │ a {
+  > 2 │     color: pink;
+      │                 
+  > 3 │     color: orange;
+      │     ^^^^^
+    4 │ }
+    5 │ 
+  
+  i Consider removing the duplicate property.
+  
+
+```
+
+```
+invalid.css:7:17 lint/nursery/noDuplicateProperties ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
+
+  ! Duplicate properties are not allowed.
+  
+     6 │ a {
+   > 7 │     color: pink;
+       │                 
+   > 8 │     color: pink;
+       │     ^^^^^
+     9 │     color: pink;
+    10 │ }
+  
+  i Consider removing the duplicate property.
+  
+
+```
+
+```
+invalid.css:8:17 lint/nursery/noDuplicateProperties ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
+
+  ! Duplicate properties are not allowed.
+  
+     6 │ a {
+     7 │     color: pink;
+   > 8 │     color: pink;
+       │                 
+   > 9 │     color: pink;
```

Comment:
The text range seems incorrect, did you notice the `>` character is on line 8? It should not.

Also, It would be nice to add a new note that shows where the first instance of the property is.